test/expect/HasDecompTest.test_has_decomposition.expect

```diff
@@ -653,6 +653,7 @@ aten::_validate_compressed_sparse_indices
 aten::_values
 aten::_values_copy
 aten::_values_copy.out
+aten::_weight_fp8_mm
```

Comment:
Even we don't have pack for fp8, it is better we can align the name with int8 to use _weight_int8pack_mm, then it is easier for user to understand that this API is for WOQ case.